app/controllers/jobs_controller.rb

```diff
@@ -39,6 +39,20 @@ def run
     end
   end
 
+  def retry_queued
+    @jobs = Delayed::Job.where('attempts > 1')
```

Comment:
We have a scope which does part of the job [Delayed::Job.awaiting_retry](https://github.com/cantino/huginn/blob/master/config/initializers/delayed_job.rb#L15)